Resources/public/js/views/fields/ez-binarybase-editview.js

```diff
@@ -357,6 +357,9 @@ YUI.add('ez-binarybase-editview', function (Y) {
          */
         _prepareDrop: function (e) {
             e.preventDefault();
+            if (this.get('isNotTranslatable')) {
```

Comment:
that's the default browser's behavior when you drop an image in a page that does not handle the drop event. And indeed, you are right, it makes sense to avoid it, so you can keep the code as it is.